bazel/devbuild/BUILD.bazel

```diff
@@ -5,14 +5,16 @@ sh_template(
     data = [
         "//bazel/release:container_sums",
         "//bootstrapper/cmd/bootstrapper:bootstrapper_linux_amd64",
+        "//cli:cli_enterprise_host",
         "//cli:cli_oss_host",
         "//debugd/cmd/cdbg:cdbg_host",
         "//upgrade-agent/cmd:upgrade_agent_linux_amd64",
     ],
     substitutions = {
         "@@BOOTSTRAPPER@@": "$(rootpath //bootstrapper/cmd/bootstrapper:bootstrapper_linux_amd64)",
         "@@CDBG@@": "$(rootpath //debugd/cmd/cdbg:cdbg_host)",
-        "@@CLI@@": "$(rootpath //cli:cli_oss_host)",
+        "@@CLI_ENTERPRISE@@": "$(rootpath //cli:cli_enterprise_host)",
```

Comment:
I think we should "guard" this behavior by a flag that must be set in e.g. the `.bazeloverwriterc` so that a 3rd party contributor must explicitly take action to enable the enterprise cli. 

Maybe to make the implementation a bit easier, I'd suggest that the flag just toggles between the `build/constellation` binary being enterprise or not.

Reply:
I'm a little bit worried about breaking developer's flows (clone, build, run) but I don't have an obvious alternative. Maybe a good middle ground is building both but naming them `constellation-oss` and `constellation-enterprise`?
(This is maybe even worse).

Reply:
As per discussion: add a flag which has not default value and must be set in the `.bazeloverwriterc` when `//:devbuild` is called.